app/views/partials/_nav.html.erb

```diff
@@ -11,11 +11,12 @@
         <%= link_to "About", about_path %>
       </div>
       <div class="align-middle table-cell p3">
-        <%= link_to "Register", register_path %>
-      </div>
-      <div class="align-middle table-cell p3">
-        <%= link_to 'Login', login_path, class: "px2"%>
-        <%= link_to 'Sign Out', logout_path, method: :delete %>
+      <% if current_user %>
+        <%= link_to 'My Garden', '/mygarden'%>
+        <%= link_to 'Sign Out', '/logout', method: :delete %>
+      <% else %>
+        <%= link_to 'Sign In', '/login', class: "px2"%>
```

Comment:
I wonder if we even need a sign in here. Maybe we just have the button to sign in with google instead of a redirect?